examples/recipes/multiple-data-sources/queries/run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# JWT decode function https://gist.github.com/a-magdy/a771f1426043ab4b66b19cc9a652908b
+_decode_base64_url() {
+  local len=$((${#1} % 4))
+  local result="$1"
+  if [ $len -eq 2 ]; then result="$1"'=='
+  elif [ $len -eq 3 ]; then result="$1"'=' 
+  fi
+  echo "$result" | tr '_-' '/+' | base64 -d
+}
+decode_jwt() { _decode_base64_url $(echo -n $1 | cut -d "." -f ${2:-2}) | jq .; }
+
+host=cube
+port=4000
+loadUrl=cubejs-api/v1/load
+readyzUrl=readyz
+
+avocadotoken=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwidGVuYW50IjoiQXZvY2FkbyBJbmMiLCJpYXQiOjE1MTYyMzkwMjIsImV4cCI6MTcyNDk5NTU4MX0.MZUdOATD7c7A6tEQDU_Iq0VI0TJUzO-PWmsFgkZ76uM
```

Comment:
I guess it should be `avocadoToken`, with a capital letter?

Reply:
Fixed